src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSource.java

```diff
@@ -535,22 +536,24 @@ protected SCMRevision retrieve(@NonNull String thingName, @NonNull TaskListener
     @NonNull
     @Override
     protected List<Action> retrieveActions(SCMSourceEvent event, @NonNull TaskListener listener) {
-        LOGGER.info(String.format("e, l..%s", Thread.currentThread().getName()));
         List<Action> result = new ArrayList<>();
         if (gitlabProject == null) {
             try {
                 GitLabApi gitLabApi = apiBuilder(serverName);
                 listener.getLogger().format("Looking up project %s%n", projectPath);
                 gitlabProject = gitLabApi.getProjectApi().getProject(projectPath);
-                result.add(new ObjectMetadataAction(null, gitlabProject.getDescription(),
```

Comment:
actions we only added once but as soon as navigated away the object metadata would be gone 😆